CHANNELS.md

```diff
@@ -0,0 +1,46 @@
+# Channels
+
+`fuelup` adopts a simplified version of `rustup` [channels](https://rust-lang.github.io/rustup/concepts/channels.html). Currently, only the `latest` channel will be published and serve as a source of distribution of Fuel toolchain binaries.
+
+The `latest` channel is possibly updated by a scheduled GitHub workflow that **runs every 30 minutes** and checks for new releases of `forc` and `fuel-core`.
+
+### How the CI updates the `latest` channel
+
+A channel is automatically updated if integration tests pass for the specific version of `forc` and `fuel-core`. This channel is typically found in a TOML file, and for the `latest` channel, is named `channel-fuel-latest.toml`.
+
+### How the workflow works, a summary
+
+The short explanation of how the scheduled workflow works is that we fetch all the latest versions after the latest published versions within `channel-fuel-latest.toml`, and we run tests for the possible combinations of them if not already marked as incompatible. After tests are done we either mark the version sets as incompatible if the tests failed or update the channel.
+
+## Developer Guide
+
+### Understanding the workflow
```

Comment:
```suggestion
### Understanding the `latest` workflow
```

Reply:
We might want to have different sections for latest and nightly, as the workflows will likely be a bit different?